beamerinnerthemeOSU.sty

```diff
@@ -7,7 +7,7 @@
 
 
 \mode<presentation>
-
+\usetikzlibrary{positioning}
```

Comment:
Let's load this in `beamerthemeOSU.sty` where we are loading the other packages.